bootstrap/interfaces/secret.go

```diff
@@ -23,6 +23,9 @@ type SecretProvider interface {
 	// Service key is use as the access token role which must have be previously setup.
 	GetAccessToken(tokenType string, serviceKey string) (string, error)
 
+	// ListSecretPaths returns a list of secret keys from an insecure/secure secrets secret store.
+	ListSecretPaths(path string) ([]string, error)
```

Comment:
This should be retuning the list of paths for the current service, not secret keys at a path

```suggestion
	ListSecretPaths() ([]string, error)
```